components/camel-pulsar/src/main/java/org.apache.camel.component.pulsar/PulsarComponent.java

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
We usually split the package path so you should have org/apache/camel/component/pulsar and not the dotted version

Reply:
yeah I suspect that this is a git merge/rename artefact - I will correct it

Reply:
done